org.jellyfin.JellyfinServer.yml

```diff
@@ -457,144 +361,40 @@ modules:
-            -Np1 -i "debs/patches/$p" -d ./3rdparty/glad; fi; done
-      - type: archive
-        url: https://deb.debian.org/debian/pool/main/j/jinja2/jinja2_3.1.2.orig.tar.gz
-        sha256: 31351a702a408a9e7595a8fc6150fc3f43bb6bf7e319770cbc0db9df9437e852
-        dest: 3rdparty/jinja
-        x-checker-data:
-          type: html
-          url: https://packages.debian.org/source/bookworm/jinja2
-          version-pattern: jinja2 \((\d+\.\d+\.\d+)
-          url-template: https://deb.debian.org/debian/pool/main/j/jinja2/jinja2_$version.orig.tar.gz
-      - type: archive
-        url: https://deb.debian.org/debian/pool/main/j/jinja2/jinja2_3.1.2-1.debian.tar.xz
-        sha256: 1b94ce8734b27981bb40d639423dce1bd4b624c17fdf457e73ef8a33cfca501a
-        dest: 3rdparty/jinja/debs
-        x-checker-data:
-          type: html
-          url: https://packages.debian.org/source/bookworm/jinja2
-          version-pattern: jinja2 \(([^\)]+)\)
-          url-template: https://deb.debian.org/debian/pool/main/j/jinja2/jinja2_$version.debian.tar.xz
-      - type: shell
-        commands:
-          - for p in $(cat 3rdparty/jinja/debs/patches/series); do if [[ $p != "#"* ]]; then patch
-            -Np1 -i "debs/patches/$p" -d ./3rdparty/jinja; fi; done
-      - type: archive
-        url: https://deb.debian.org/debian/pool/main/m/markupsafe/markupsafe_2.1.2.orig.tar.gz
-        sha256: abcabc8c2b26036d62d4c746381a6f7cf60aafcc653198ad678306986b09450d
-        dest: 3rdparty/markupsafe
-        x-checker-data:
-          type: html
-          url: https://packages.debian.org/source/bookworm/markupsafe
-          version-pattern: markupsafe \((\d+\.\d+\.\d+)
-          url-template: https://deb.debian.org/debian/pool/main/m/markupsafe/markupsafe_$version.orig.tar.gz
-      - type: archive
-        url: https://deb.debian.org/debian/pool/main/m/markupsafe/markupsafe_2.1.2-1.debian.tar.xz
-        sha256: 1165a4439404a0a34d9e2d533b5f8750567a3e4588337d4e25796b8525262f76
-        dest: 3rdparty/markupsafe/debs
-        x-checker-data:
-          type: html
-          url: https://packages.debian.org/source/bookworm/markupsafe
-          version-pattern: markupsafe \(([^\)]+)\)
-          url-template: https://deb.debian.org/debian/pool/main/m/markupsafe/markupsafe_$version.debian.tar.xz
-      - type: shell
-        commands:
-          - for p in $(cat 3rdparty/markupsafe/debs/patches/series); do if [[ $p != "#"* ]]; then patch
-            -Np1 -i "debs/patches/$p" -d ./3rdparty/markupsafe; fi; done
+          type: git
+          tag-pattern: ^vulkan-sdk-([\d.]+)$
+      # Already included as a submodule: glad, jinja, markupsafe
 
   - name: numactl
-    buildsystem: autotools
+    #buildsystem: autotools
+    buildsystem: simple
+    build-commands:
+      - ./autogen.sh
+      - ./configure --prefix $FLATPAK_DEST
+      - make -j$FLATPAK_BUILDER_N_JOBS PREFIX=$FLATPAK_DEST install
```

Comment:
this should use autotools buildsystem

Reply:
I get this error message when I try your suggested changes on a local build:

```
configure: error: cannot find required auxiliary files: compile config.guess config.sub ltmain.sh missing install-sh
```

I can see that:

- Handbrake is using it in a different way: https://github.com/flathub/fr.handbrake.ghb/blob/master/fr.handbrake.ghb.json
- Sunshine uses an older version from Debian: https://github.com/flathub/dev.lizardbyte.app.Sunshine/blob/master/dev.lizardbyte.app.Sunshine.yml

I opt for the way Handbrake does it adding `rm-configure: true` leaving `#buildsystem: autotools` and dropping the other lines.

Reply:
> Handbrake is using it in a different way: https://github.com/flathub/fr.handbrake.ghb/blob/master/fr.handbrake.ghb.json

this one is fine, if nothing is specified it defaults to autotools.